scrapy/utils/log.py

```diff
@@ -144,10 +144,7 @@ def _get_handler(settings):
 def log_scrapy_info(settings):
     logger.info("Scrapy %(version)s started (bot: %(bot)s)",
                 {'version': scrapy.__version__, 'bot': settings['BOT_NAME']})
-    logger.info("Versions: %(versions)s",
-                {'versions': ", ".join("%s %s" % (name, version)
-                    for name, version in scrapy_components_versions()
-                    if name != "Scrapy")})
+    print(*scrapy_components_versions(), sep = "\n")
```

Comment:
Wouldn't this be Python 3 syntax only? I don't see a `from __future__ import print_function` at the top of this file, is it somewhere else?

Reply:
Yes , I forgot to take care about Python 2. I will commit the changes. Thanks :)!

Reply:
This might be a good time to delete the spaces around the equals sign too.

```python
print(*scrapy_components_versions(), sep="\n")
```

Reply:
But why this checks were not successful? @42B